test/cmdlineTests/ir_compiler_subobjects/err

```diff
@@ -0,0 +1,5 @@
+Warning: Unused local variable.
+ --> ir_compiler_subobjects/input.sol:6:9:
+  |
+6 |         C c = new C();
```

Comment:
It would maybe be good to avoid this, but i'll merge it anyway. this PR is rather old already.